addons/pos_self_order/models/pos_order.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+from typing import Dict
+
+from odoo import models
+
+
+class PosOrderLine(models.Model):
+    _inherit = "pos.order.line"
+
+    def _export_for_ui(self, orderline):
+        return {
+            "uuid": orderline.uuid,
+            "note": orderline.note,
+            **super()._export_for_ui(orderline),
```

Comment:
```suggestion
            **super()._export_for_ui(orderline),
            "uuid": orderline.uuid,
            "note": orderline.note,
```

I'm guessing you want to override the "uuid" and "note" coming from the original result of `_export_for_ui`. If so, you should consider this suggestion.